test/integration/emitter_test.cpp

```diff
@@ -612,6 +612,38 @@ TEST_F(EmitterTest, SimpleGlobalSettings) {
   ExpectEmit("-   ? key 1\n    : value 1\n    ? key 2\n    : [a, b, c]");
 }
 
+TEST_F(EmitterTest, GlobalLongKeyOnSeq) {
+  out.SetMapFormat(LongKey);
+
+  out << BeginMap;
+  out << Key << Anchor("key");
+  out << BeginSeq << "a"
+      << "b" << EndSeq;
+  out << Value << Anchor("value");
+  out << BeginSeq << "c"
+      << "d" << EndSeq;
+  out << Key << Alias("key") << Value << Alias("value");
+  out << EndMap;
+
+  ExpectEmit("? &key\n  - a\n  - b\n: &value\n  - c\n  - d\n? *key\n: *value");
```

Comment:
Can you use raw literals here? I know the other examples don't do this, but they were from before that feature was widely accepted. I think it would be more readable here.

Reply:
Do you mean raw string literal?  I try it
```
  ExpectEmit(
R"(? &key
  - a
  - b
: &value
  - c
  - d
? *key
: *value)");
```
haha, It is intuitional, but the indents and alignment look like not confortable. 
```
  ExpectEmit("? &key\n"
             "  - a\n"
             "  - b\n"
             ": &value\n"
             "  - c\n"
             "  - d\n"
             "? *key\n"
             ": *value");
```
This format is both readable and intuitional, but the escape char is not good.
What's you opinoin about this?

Reply:
I’d go with the R style. The first line doesn’t line up right, but subsequent ones do and that’s probably fine.